tracer/src/Datadog.Trace/Telemetry/DTOs/LogMessageData.cs

```diff
@@ -0,0 +1,49 @@
+﻿// <copyright file="LogMessageData.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+using System;
+using System.Text;
+using Datadog.Trace.Vendors.Newtonsoft.Json.Converters;
+
+namespace Datadog.Trace.Telemetry.DTOs;
+
+internal class LogMessageData
+{
+    private const int FixedSerializationCharacters =
```

Comment:
Super-nitty, but consider renaming this field for clarity. It took more more than 5 seconds (and looking at usage in below) to realize it was a _count_ of characters.